content/en/docs/tasks/tools/install-kubectl-windows.md

```diff
@@ -222,7 +222,23 @@ Below are the procedures to set up autocompletion for PowerShell.
    ```powershell
    del kubectl-convert.exe kubectl-convert.exe.sha256
    ```
+### Uninstalling kubectl on windows
 
+ {{< tabs name="kubectl_win_uninstall" >}}
```

Comment:
Similar to previous file, it would be nice to maintain consistency by aligning the order of uninstallation commands with the installation section.

Currently, the option to uninstall when curl was used for installation is missing (uninstall would be simply deleting the downloaded binary using 'del kubectl.exe kubectl.exe.sha256'). Additionally, it would be nice to include the uninstallation process using `winget`.